js/main.js

```diff
@@ -26,4 +26,102 @@ $form.addEventListener('submit', function (event) {
 
   $form.reset();
   $newEntryImg.setAttribute('src', 'images/placeholder-image-square.jpg');
+
+  $noEntries.remove();
+  i = 0;
+  $entryGallery.prepend(addEntry());
+
+  for (i = 0; i < allViews.length; i++) {
+    if (allViews[i].getAttribute('data-view') === 'entries') {
+      allViews[i].className = 'view';
+    } else {
+      allViews[i].className = 'view hidden';
+    }
+  }
+});
+
+var $body = document.querySelector('body');
+var allViews = document.querySelectorAll('.view');
+
+$body.addEventListener('click', clickHandler);
+function clickHandler(event) {
+  if (!event.target.matches('.swap')) {
+    return;
+  }
+  event.preventDefault();
+  var btnDataView = event.target.getAttribute('data-view');
+
+  for (i = 0; i < allViews.length; i++) {
+    if (allViews[i].getAttribute('data-view') === btnDataView) {
+      allViews[i].className = 'view';
+      data.view = btnDataView;
+    } else {
+      allViews[i].className = 'view hidden';
+    }
+  }
+}
+
+//   <li>
+//     <div class="row justifySpaceBetween">
+//       <div class="clmHalf">
+//         <img class="historyImg"
+//         src="https://image.shutterstock.com/image-photo/man.jpeg" alt="">
+//       </div>
+//       <div class="clmHalf">
+//         <h3>Pasley is different than Parsley</h3>
+//         <p>Dogs have smelled since the dawn of time, when they first learned exist.</p>
+//       </div>
+//     </div>
+//   </li>
+
+var $entryGallery = document.querySelector('.entryGallery');
+var $noEntries = document.querySelector('.noEntries');
+
+function addEntry() {
+  var $li = document.createElement('li');
+  var $row = document.createElement('div');
+  $row.className = 'row justifySpaceBetween';
+
+  $li.appendChild($row);
+
+  var clmHalf1 = document.createElement('div');
+  clmHalf1.className = 'clmHalf';
+  var $img = document.createElement('img');
+  $img.className = 'historyImg';
+  $img.setAttribute('src', data.entries[i].photoURL);
+
+  clmHalf1.appendChild($img);
+  $row.appendChild(clmHalf1);
+
+  var clmHalf2 = document.createElement('div');
+  clmHalf2.className = 'clmHalf';
+  var $title = document.createElement('h3');
+  $title.textContent = data.entries[i].title;
+  var $notes = document.createElement('p');
+  $notes.textContent = data.entries[i].notes;
+
+  clmHalf2.appendChild($title);
+  clmHalf2.appendChild($notes);
+  $row.appendChild(clmHalf2);
+
+  return $li;
+}
```

Comment:
You would be better off making this function receive a parameter which will be the object for a single entry. That way you can pass the entry object as an argument whenever you need to make a new entry DOM tree without having to worry about the value of `i`